src/components/ListToolbox/ListToolbox.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import { Button } from '../Buttons/Button';
+import styles from './ListToolbox.scss';
+
+type Selection = {
+    amount: number;
+    label: string;
+};
+
+type Toggle = {
+    label: string;
```

Comment:
what do you think about assigning to `label` type `SingleReactNode` for giving label more flexibility?
For example, It allows us to have icons a label if it will be needed in the future:
```
label: <>
   <Icon type="import"> Import
</>
```
Or for implementing localization:
```
label: <Localized id={fieldset.fieldsetLabelI18nKey} />
```

Reply:
I think it's a good idea